test/test_managercli.py

```diff
@@ -1646,19 +1646,18 @@ def test_insecure(self):
         self.cc.main(["--insecure"])
 
     def test_org_requires_list_error(self):
+        self.cc.is_registered = Mock(return_value=False)
```

Comment:
Hmm this is not present in main, why was it added on 1.28 only?

Reply:
It is kind of unnecessary here, the validation never gets to the part where registration is checked. For completeness it can stay here, but in that case it was forgotten in the PR to the main branch.

Reply:
OK. My note was about consistency between the two branches, otherwise if feels that either something in missing, or it is needed only in one branch (and both would require an explanation on the reason).

Either works for me:
- removing this line from this version
- adding it in `main`, and adding it to this PR as own cherry-picked commit

Reply:
Since it's not needed for the test to succeed, I'll remove it. It will be consistent with the rest of the tests -- if they check the behaviour but do not require registration status, they don't have the is_registered mock.

Reply:
OK :-)